### ISSUE TITLE
Support for contract call with reference inputs

### ISSUE THREAD

**Issue by satran004** [bloxbean/aiken-java-binding#1]
In v0.0.2, if transaction has reference inputs, script evaluation fails with following error.

```
RedeemerError { tag: "Spend", index: 0, err: ResolvedInputNotFound }
``` 

**Workaround :** During the transaction cost evaluation, remove the reference inputs from the transaction and add the plutus script to the witness set. After calculating the script cost, you can add back the reference input again.

**Comment by matiwinnetou** [comment]
Isn't already done in CCL 0.5-alpha?

**Comment by satran004** [comment]
Yes, it is included in 0.5.0-alpha.2. However, due to the package renaming in Cardano Client Lib, it is required to update the Aiken Java Binding accordingly

**Comment by satran004** [comment]
v0.0.7 supports this.
Example: https://github.com/bloxbean/aiken-java-binding/blob/main/src/integrationTest/java/com/bloxbean/cardano/aiken/tx/evaluator/GuessSumContractIntegrationTest.java
